From eb53094f171a3833f19b20667bda236d78a44ca9 Mon Sep 17 00:00:00 2001
From: Thilo Graf <dbt@novatux.de>
Date: Sun, 24 Jul 2022 14:56:09 +0200
Subject: [PATCH 2/2] openembedded: disable openembedded-layer, meta-phyton

---
 meta-networking/conf/layer.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meta-networking/conf/layer.conf b/meta-networking/conf/layer.conf
index bd0a44cae..dce550fe2 100644
--- a/meta-networking/conf/layer.conf
+++ b/meta-networking/conf/layer.conf
@@ -14,8 +14,8 @@ BBFILE_PRIORITY_networking-layer = "5"
 LAYERVERSION_networking-layer = "1"
 
 LAYERDEPENDS_networking-layer = "core"
-LAYERDEPENDS_networking-layer += "openembedded-layer"
-LAYERDEPENDS_networking-layer += "meta-python"
+#LAYERDEPENDS_networking-layer += "openembedded-layer"
+#LAYERDEPENDS_networking-layer += "meta-python"
 
 LAYERSERIES_COMPAT_networking-layer = "gatesgarth"
 
-- 
2.35.1

